tests/ui/notifications/test_notifications_api.py

```diff
@@ -35,9 +38,8 @@
 
 
 def test_notification_create_and_es_mapping(
```

Comment:
Can you split this test in two parts: one for the mapping and an other one for the creation?

Reply:
This code is from Aly, have to ask him!

Reply:
Merged these two functions in one to avoid duplicating the test of the notification creation.